src/test/resources/application-test.yml

```diff
@@ -6,7 +6,7 @@ management.endpoint:
   info.cache.time-to-live: 0
 
 spring:
-  redis.port: 6380
+  data.redis.port: 6380
```

Comment:
although this was a deprecated property warning - didn't work until I changed it to the new location